EIPS/eip-2390.md

```diff
@@ -0,0 +1,316 @@
+---
+eip: 2390
+title: Geo-ENS
+author: James Choncholas (@james-choncholas)
+discussions-to: https://github.com/ethereum/EIPs/issues/2959
+status: Draft
+type: Standards Track
+category: ERC
+created: 2019-11-15
+requires: 137, 165, 1062, 1185
+---
+
+## Simple Summary
+GeoENS brings geographic split horizon capabilities to ENS. It's GeoDNS for ENS!
+
+## Abstract
+This EIP specifies an ENS resolver interface for geographically split horizon DNS.
+Geographic split horizon DNS returns resource records that are specific to an end
+user's location.
+This technique is commonly used by CDNs to direct traffic to content caches nearest users.
+Geographic split horizon resolution is primarily geared towards ENS
+resolvers storing DNS resource records [EIP-1185](./eip-1185.md), although the technique could be
+used on other interfaces like IPFS content hash storage [EIP-1162](./eip-1162.md).
```

Comment:
There is no EIP-1162.  Perhaps you meant some other EIP?

Reply:
Nice catch! Was supposed to be EIP 1062 - fixed in 0d84edfb88fa1cf706ea5e1fddfd7f6adb7c51ce